### ISSUE TITLE
'Expression does not conform to type NILLiteralConvertible' in Swift 2.0 branch in Xcode 7 beta 5

### ISSUE THREAD

**Issue by rajohns08** [Alamofire/Alamofire#702]
Manager.swift for the following line:

```swift
if CFStringTransform(mutableUserAgent, nil, transform, 0) == 1 {
    return mutableUserAgent as String
}
```

The `nil` parameter passed to `CFStringTransform` is reporting the error.

I didn't see this issue until right after upgrading to beta 5.

**Comment by rajohns08** [comment]
See https://github.com/Raizlabs/BonMot/issues/55 and https://developer.apple.com/library/prerelease/ios/documentation/CoreFoundation/Reference/CFMutableStringRef/index.html#//apple_ref/c/func/CFStringTransform

**Comment by rajohns08** [comment]
Proposed fix is:

```swift
if CFStringTransform(mutableUserAgent, UnsafeMutablePointer<CFRange>(), transform, false) {
    return mutableUserAgent as String
}
```

But I can't be 100% sure what returning 1 used to mean from CFStringTransform (although I'm assuming it meant true).

**Comment by rajohns08** [comment]
Ok this has already been fixed with [commit 62947d6](https://github.com/Alamofire/Alamofire/commit/62947d6deffc30d3128da580d91084a174fe485d) on August 6th. Why hasn't that change made it to cocoapods yet?

**Comment by rajohns08** [comment]
closing because I just realized `pod install` does not update pods
